bootstrap.sh

```diff
@@ -29,6 +29,6 @@ if [ ! -z "$XCS_BOT_ID"  ]; then
 fi
 
 if [ ! -f Carthage/Cartfile.resolved ]; then
-  carthage bootstrap $CARTHAGE_VERBOSE --platform ios --color auto --no-use-binaries
+  carthage bootstrap $CARTHAGE_VERBOSE --platform ios --color auto
```

Comment:
Note @sleroux that we asked them for the binaries and they prefer we use them rather than compile. They are currently working out the situation with Fennec Android to allow Taipei team to compile from source though. So if we want to compile from source we probably can - let me know.